packages/common/lib/common/tools.js

```diff
@@ -3439,9 +3439,36 @@ async function getInstancesOrderedByStartPrio(objects, logger, logPrefix = '') {
  * @returns {Promise<void>}
  */
 async function setExecutableCapabilities(execPath, capabilities, modeEffective, modePermitted, modeInherited) {
+    if (!Array.isArray(capabilities) || !capabilities.length) {
+        throw new Error('No capabilities array provided');
+    }
+
     // if not linux do nothing and silently exit
     if (os.platform() === 'linux') {
-        if (Array.isArray(capabilities) && capabilities.length) {
+        // if Docker and Admin Capabilities should be set check if we are allowed to do that
+        if (isDocker() && capabilities.includes('cap_net_admin')) {
+            try {
+                const systemCaps = fs.readFileSync(`/proc/${process.pid}/status`, 'utf-8');
```

Comment:
I'm not sure about this:
> $$ will return the PID of your process so when typed into a terminal window it will return the Process ID of the shell you are in

@buanet do you know if we have to determine the init script's PID somehow?

Reply:
In the installer we also use "our own" process id which is the bash one ... should work

Reply:
Also in example that @buanet sent me it was a bash process

Reply:
Yeah I guess this doesn't matter, but I'm not sure.

Reply:
I think the process doesn't matter... Seems like CapBnd looks the same for every process...

```
root@iobroker:/opt/iobroker# grep ^CapBnd /proc/166/status
CapBnd: 00000000a80425fb
root@iobroker:/opt/iobroker# grep ^CapBnd /proc/203/status
CapBnd: 00000000a80425fb
root@iobroker:/opt/iobroker# grep ^CapBnd /proc/1/status
CapBnd: 00000000a80425fb
```